projects/bp-gallery/src/components/views/exhibitions/ExhibitionTool.tsx

```diff
@@ -0,0 +1,351 @@
+          <SortableContext items={itemIds}>
+            {dragItems?.map(item =>
+              item.id === activeId ? (
+                <div key={item.id} className='opacity-25'>
+                  {item.sortableElement}
+                </div>
+              ) : (
+                item.sortableElement
+              )
+            )}
+          </SortableContext>
+        </DndContext>
+      ) : (
+        <>{dragItems?.map(item => item.element)}</>
+      )}
+    </div>
+  );
+};
+
+const TitleDropzone = () => {
+  const titlePicture = useContext(ExhibitionGetContext).getTitlePicture();
+  const { setNodeRef } = useDroppable({ id: 'titleDropzone' });
+  return (
+    <div
+      ref={setNodeRef}
+      className='h-[10rem] overflow-hidden w-[12rem] border-solid rounded-xl p-2 box-border flex flex-wrap gap-2 bg-gray-200'
+    >
+      {titlePicture?.element}
+    </div>
+  );
+};
+
+const ExhibitionManipulator = () => {
+  const { t } = useTranslation();
+  const sections = useContext(ExhibitionGetContext).getAllSections();
+  const addSection = useContext(ExhibitionSectionUtilsContext).addSection;
+  const scroll = useRef(0);
+  const scrollDivRef = useRef<HTMLDivElement | null>(null);
+
+  //TODO
+  //@ts-ignore
+  const handleScroll = e => {
+    if (e.target.scrollTop !== 0) scroll.current = Number(e.target.scrollTop);
+  };
+
+  useEffect(() => scrollDivRef.current?.scrollTo(0, scroll.current));
+  return (
+    <div className='flex flex-col items-stretch h-full w-full'>
+      <div className='absolute z-[999] right-7 top-[6rem]'>
+        <PublishButton />
+      </div>
+      <div className='text-xl'>Ausstellungstool</div>
+      <div
+        className='border-solid flex-1 p-2 overflow-y-auto'
+        ref={scrollDivRef}
+        onScroll={handleScroll}
+      >
+        <Introduction />
+        {sections?.map(section => (
+          <Section key={section.id} id={section.id} />
+        ))}
+        <div className='grid place-content-center'>
+          <Button onClick={addSection} variant='outlined'>
+            {t('exhibition.manipulator.section.add-section')}
+          </Button>
+        </div>
+        <EndCard />
+      </div>
+    </div>
+  );
+};
+
+const Introduction = () => {
+  const { t } = useTranslation();
+  const { getTitle, getIntroduction } = useContext(ExhibitionGetContext);
+
+  const { setTitle, setIntroduction } = useContext(ExhibitionSetContext);
+
+  const extraOptions = {
+    height: 300,
+    allowReziseX: false,
+    allowReziseY: false,
+    preset: undefined,
+    placeholder: t('exhibition.manipulator.intro.text-placeholder'),
+    statusbar: false,
+    tabIndex: 0,
+    className: 'z-0',
+  };
+  return (
+    <div className='flex flex-col flex-1 gap-2'>
+      <TextField
+        className='flex-1'
+        variant='standard'
+        placeholder={t('exhibition.manipulator.intro.title-placeholder')}
+        value={getTitle()}
+        onChange={event => setTitle(event.target.value)}
+      />
```

Comment:
Maybe also extract this part into an extra component then, to avoid unneccassary rerenders effecting the rest of this component (`TitleDropzone`, `TextEditor`).
```const ExhibitionTitleInput = () => {
  const { t } = useTranslation();
  const { getTitle } = useContext(ExhibitionGetContext);

  const { setTitle } = useContext(ExhibitionSetContext);

  const [currentTitle, setCurrentTitle] = useState<string>(getTitle());

  return (
    <TextField
      className='flex-1'
      variant='standard'
      placeholder={t('exhibition.manipulator.intro.title-placeholder')}
      value={currentTitle}
      onChange={event => setCurrentTitle(event.target.value)}
      onBlur={_ => setTitle(currentTitle)}
    />
  );
};
```